framework/src/util/SharedLibrary.cpp

```diff
@@ -124,6 +123,21 @@ void SharedLibrary::Load(int flags)
 #endif
 }
 
+void SharedLibrary::Load(const std::unordered_map<std::string, Any>& config)
+{
+#ifdef US_PLATFORM_POSIX
+  int options = RTLD_LAZY | RTLD_LOCAL;
+  try {
+    options = any_cast<int>(config.at(Constants::LIBRARY_LOAD_OPTIONS));
+  } catch (...) {
```

Comment:
I think it's better to use the `[]` API instead of the `at` since the option is guaranteed to exist based on what I understand from `InitProperties`.  If that is not true, use `find` and avoid exceptions from `at`

As far as the exceptions from any_cast, we should log the exception instead of swallowing it.

Reply:
Done.

Reply:
I was unable to change this to use the [] API instead of **at** as the config object is const, and the [] API is not, while the **at** API is.